js/modules/sliderOurVisitors.js

```diff
@@ -0,0 +1,119 @@
+const slidesContainer = document.getElementById("ourVisitors__photo_container");
+const slidesQuantity = slidesContainer.querySelectorAll(".ourVisitors__photo").length;
+let slideWidth = slidesContainer.querySelector(".ourVisitors__photo").offsetWidth;
+let sliderMinPossiblePosition = -(slidesQuantity - 1)*slideWidth;
+const SLIDER_MAX_POSSIBLE_POSITION = 0;
+let sliderCurrentShift = 0;
+let sliderCurrentPosition;
+
+const sliderRangeWidth = document.getElementById("slider_range").offsetWidth;
+const sliderThumb = document.getElementById("slider_thumb");
+const sliderThumbWidth = sliderRangeWidth/slidesQuantity;
+const SLIDER_THUMB_MIN_POSSIBLE_POSITION = 0;
+const sliderThumbMaxPossiblePosition = (slidesQuantity - 1)*sliderThumbWidth;
+let sliderThumbCurrentShift = 0;
+
+const btnPrevOurVisitors = document.getElementById("btn_prev_ourVisitors");
+const btnNextOurVisitors = document.getElementById("btn_next_ourVisitors");
+
+sliderThumb.style.width = sliderThumbWidth + "px"; 
+
+const mql320 = window.matchMedia("(min-width: 320px)");
```

Comment:
Think over the idea of using percent in transform and maybe you won't need mediaqueries at all

Reply:
Done.